### PR TITLE
[OV] Disable FQ propagation through ReLU

### PR DESCRIPTION
**Pull request by l-bat**
### Reason for changes

To align NNCF quantization with POT

### Related tickets

105079

### CODE REVIEW

**Comment by l-bat** [comment]
run onnx pre-commit tests

**Comment by l-bat** [comment]
run openvino pre-commit tests

File: nncf/experimental/openvino_native/quantization/default_quantization.py [hunk 1]
```diff
@@ -58,6 +58,8 @@
         ov_metatypes.OVExpMetatype,
         ov_metatypes.OVLogMetatype,
         ov_metatypes.OVAssignMetatype,
+        ov_metatypes.OVReluMetatype,
+        ov_metatypes.OVDeformableConvolutionMetatype,
```
Comment (alexsu52) [inline nncf/experimental/openvino_native/quantization/default_quantization.py:62]: Please, add the same changes to ONNX, Torch and TensorFlow backends.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

**Comment by l-bat** [comment]
e2e_pytorch: 272
e2e_onnx: 291
sota_eval_tensorflow: 302
conformance: 92

**Comment by l-bat** [comment]
run pytorch pre-commit tests

**Comment by l-bat** [comment]
run tensorflow pre-commit tests

**Comment by l-bat** [comment]
run onnx pre-commit tests

File: nncf/experimental/openvino_native/quantization/default_quantization.py (hunk 1 above)
Comment (l-bat) [reply nncf/experimental/openvino_native/quantization/default_quantization.py:62]: Added

**Comment by alexsu52** [comment]
> e2e_pytorch: 272 e2e_onnx: 291 sota_eval_tensorflow: 302 conformance: 92

Could you provide a summary of the results?

**Review by alexsu52 — COMMENTED** [review]
LGTM

**Pull request merged**